app/controllers/api/v0/articles_controller.rb

```diff
@@ -85,16 +85,16 @@ def article_params
           :title, :body_markdown, :published, :series,
           :main_image, :canonical_url, :description, tags: []
         ]
-        allowed_params << :organization_id if params["article"]["organization_id"] && allowed_to_change_org_id?
+        allowed_params << :organization_id if params.dig("article", "organization_id") && allowed_to_change_org_id?
```

Comment:
Cool, had no idea about this method. Really useful.

Reply:
TIL!